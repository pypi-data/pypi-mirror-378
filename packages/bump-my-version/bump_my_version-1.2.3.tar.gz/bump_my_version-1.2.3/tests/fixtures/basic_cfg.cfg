[options.packages.find]
exclude =
	example*
	tests*
	docs*
	build

[bumpversion]
commit = True
tag = True
current_version = 1.0.0
parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\-(?P<release>[a-z]+))?
serialize =
	{major}.{minor}.{patch}-{release}
	{major}.{minor}.{patch}

[darglint]
ignore = DAR402

[bumpversion:file:setup.py]

[bumpversion:file:bumpversion/__init__.py]

[bumpversion:file:CHANGELOG.md]
search = **unreleased**
replace = **unreleased**
	**v{new_version}**

[bumpversion:part:release]
optional_value = gamma
values =
	dev
	gamma
