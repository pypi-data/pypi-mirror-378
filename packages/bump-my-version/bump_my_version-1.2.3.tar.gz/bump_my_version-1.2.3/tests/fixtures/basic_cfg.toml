[tool.pytest.ini_options]
norecursedirs = [
    ".*",
    "build",
    "dist",
    "{arch}",
    "*.egg",
    "venv",
    "requirements*",
    "lib",
]
python_files = "test*.py"
addopts = [
    "--cov=bumpversion",
    "--cov-branch",
    "--cov-report=term",
    "--cov-report=html",
]

[tool.bumpversion]
commit = true
tag = true
current_version = "1.0.0"
parse = "(?P<major>\\d+)\\.(?P<minor>\\d+)\\.(?P<patch>\\d+)(\\-(?P<release>[a-z]+))?"
serialize = [
    "{major}.{minor}.{patch}-{release}",
    "{major}.{minor}.{patch}"
]
[[tool.bumpversion.files]]
filename = "setup.py"

[[tool.bumpversion.files]]
filename = "bumpversion/__init__.py"

[[tool.bumpversion.files]]
filename = "CHANGELOG.md"
search = "**unreleased**"
replace = """**unreleased**
**v{new_version}**"""

[tool.bumpversion.parts.release]
optional_value = "gamma"
values =[
    "dev",
    "gamma",
]

[tool.othertool]
bake_cookies = true
ignore-words-list = "sugar, salt, flour"
