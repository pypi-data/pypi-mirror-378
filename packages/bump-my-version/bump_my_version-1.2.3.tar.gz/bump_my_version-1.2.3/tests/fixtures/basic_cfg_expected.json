{
  "commit": true,
  "tag": true,
  "current_version": "1.0.0",
  "parse": "(?P<major>\\d+)\\.(?P<minor>\\d+)\\.(?P<patch>\\d+)(\\-(?P<release>[a-z]+))?",
  "serialize": [
    "{major}.{minor}.{patch}-{release}",
    "{major}.{minor}.{patch}"
  ],
  "files": [
    {
      "filename": "setup.py"
    },
    {
      "filename": "bumpversion/__init__.py"
    },
    {
      "filename": "CHANGELOG.md",
      "search": "**unreleased**",
      "replace": "**unreleased**\n**v{new_version}**"
    }
  ],
  "parts": {
    "release": {
      "optional_value": "gamma",
      "values": [
        "dev",
        "gamma"
      ]
    }
  }
}
